pyodide-build/pyodide_build/pyzip.py

```diff
@@ -5,7 +5,7 @@
 
 from ._py_compile import _compile
 
-# This files are removed from the stdlib by default
+# These files are removed from the stdlib by default
```

Comment:
I'd also remove "by default" here and in the second comment since it's unclear what it means.
```suggestion
# These files are removed from the stdlib
```